src/gc/gc.cpp

```diff
@@ -14772,8 +14817,9 @@ int gc_heap::generation_to_condemn (int n_initial,
         for (i = (temp_gen+1); i <= n_time_max; i++)
         {
             dynamic_data* dd = dynamic_data_of (i);
-            if ((now > dd_time_clock(dd) + power (10, i)*1000) &&
-                (dd_gc_clock (dd0) > (dd_gc_clock (dd) + (power (10, i)))) &&
+            static_data* sdata = dd->sdata;
```

Comment:
> dd->sdata; [](start = 32, length = 11)

I would add something like dd_time_clock_interval and dd_gc_clock_interval in gcpriv.h to make it consistent with the rest of the dynamic_data usae.

Reply:
Fixed